File: python/program/InformasiKontak.py
# kelas induk
class InformasiKontak:
    
    def __init__(self, noHp: str, email: str, alamat: str):
        self._noHp = str(noHp)       # semi private
        self._email = str(email)     # semi private
        self._alamat = str(alamat)   # semi private

    # Getter dan Setter No HP
    def getNoHp(self) -> str:
        return self._noHp
    
    def setNoHp(self, noHp: str) -> None:
        self._noHp = str(noHp)

    # Getter dan Setter Email
    def getEmail(self) -> str:
        return self._email
    
    def setEmail(self, email: str) -> None:
        self._email = str(email)

    # Getter dan Setter Alamat
    def getAlamat(self) -> str:
        return self._alamat
    
    def setAlamat(self, alamat: str) -> None:
        self._alamat = str(alamat)

File: java/program/InformasiKontak.java
public class InformasiKontak {
    // Di sini saya bikin class namanya InformasiKontak.
    // Class ini dipakai buat nyimpen data kontak seperti nomor HP, email, sama alamat.
    private String noHp;
    private String email;
    private String alamat;

    /*Ini constructor. Jadi waktu kita bikin objek InformasiKontak,
     otomatis kita bisa langsung masukin nilai nomor HP, email, dan alamatnya. */
    public InformasiKontak(String noHp, String email, String alamat) {
        this.noHp = noHp;
        this.email = email;
        this.alamat = alamat;
    }

    /* Getter sama setter di bawah ini fungsinya buat ngambil dan ngubah data.
     Misalnya kita mau lihat nomor HP yang udah disimpan, kita panggil getNoHp().
     Kalau mau ganti nomor HP, kita pakai setNoHp(). */
    public String getNoHp() {
        return noHp;
    }

    public void setNoHp(String noHp) {
        this.noHp = noHp;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getAlamat() {
        return alamat;
    }

    public void setAlamat(String alamat) {
        this.alamat = alamat;
    }
}
